src/main/java/io/outfoxx/swiftpoet/ParameterizedTypeName.kt

```diff
@@ -102,3 +102,6 @@ class ParameterizedTypeName internal constructor(
 
 fun DeclaredTypeName.parameterizedBy(vararg typeArguments: TypeName) =
   ParameterizedTypeName(null, this, typeArguments.toList())
+
```

Comment:
Unrelated change. Please open a new PR. I will say this, that I view this as a pre-optimization. The performance difference here is very small.